platform/default/src/mbgl/storage/offline_database.cpp

```diff
@@ -76,30 +76,36 @@ void OfflineDatabase::changePath(const std::string& path_) {
     initialize();
 }
 
-void OfflineDatabase::cleanup() {
+void OfflineDatabase::cleanup(bool deleteDBFile) {
     // Deleting these SQLite objects may result in exceptions
     try {
         statements.clear();
         db.reset();
+        if (deleteDBFile)
```

Comment:
I would separate deletion and cleanup into separate methods.

Reply:
Done!